crates/oxc_parser/src/ts/types.rs

```diff
@@ -1403,7 +1407,7 @@ impl<'a> ParserImpl<'a> {
             Kind::Import => {
                 matches!(self.peek_kind(), Kind::LParen | Kind::LAngle | Kind::Dot)
             }
-            kind => kind.is_identifier(),
+            _ => false,
```

Comment:
Unlike `TypeScript`, this function only used in `is_start_of_expression`, and it has the same check, so this can be removed here